apps/client/app/theme/typography.ts

```diff
@@ -0,0 +1,72 @@
+// 値部分をリテラルとして吐いてほしいので https://github.com/microsoft/TypeScript/issues/32063 が解決されるまでは ts ファイルとして置く
+// json への移行を簡単にするために、json レベルの記法のみにする
+
+import type { CSSProperties } from "react";
+
+export default {
+  /**
+   * font-size: 32px
+   */
+  heading1: {
+    fontSize: "2rem",
```

Comment:
```suggestion
  heading1: {
    fontSize: "2rem", // 32px
```
とかで良さそうだけどなんかの記法だったりする？

Reply:
JSDoc 形式で heading1 の部分に書くと
`typography.heading1` って書いた部分にホバーすると px が見れる